setup.cfg

```diff
@@ -50,7 +50,7 @@ install_requires =
     # bleeding-edge version.
     # NB: no graphene version specified; we only make light use of it in our
     # own code, so graphene-tornado's transitive version should do.
-    cylc-flow>=8.1.2, <8.2
+    cylc-flow>=8.1.3, <8.2
```

Comment:
(This seems to be incompatible with the sibling cylc-flow PR, I had to change it back to get cylc-uiserver to pip install on this branch).

Reply:
Ah, would probably need to be `>=8.1.3.dev` until released.